src/documents/IoCTestingFramework.md

```diff
@@ -4,25 +4,30 @@ layout: default
 category: Infusion
 ---
 
-The IoC Testing Framework is written using our existing jqUnit wrapper for jQuery's [QUnit](http://qunitjs.com/) as a base.
+The IoC Testing Framework is written using our existing [jqUnit](jqUnit.md) wrapper for jQuery's [QUnit](http://qunitjs.com/) as a base.
 The IoC Testing Framework is both written using Fluid's [IoC](HowToUseInfusionIoC.md) system, as well as being designed to test components
 which are themselves written using IoC. This framework aims to extend our power to write tests in various directions at the same time.
 As well as creating an idiomatic way of writing *integration tests* addressed at realistic-sized chunks of applications, expressed as IoC component
 trees, the IoC testing framework also considerably eases the task of testing *complex event sequences* - that is, sequences of application state
 that are derived from an alternating conversation between user interaction and application response.
 
-<div class="infusion-docs-note"><strong>Note:</strong> The IoC Testing framework is primarily for integration testing requiring asynchrony, i.e. testing that involves either.</div>
+## When to use the IoC Testing Framework ##
+
+<div class="infusion-docs-note"><strong>Note:</strong> The IoC Testing framework is primarily for integration testing requiring asynchrony, i.e. testing that involves either:
 
 1. user interaction via the DOM, or
 2. AJAX requests.
```

Comment:
Because this list is inside of a markup block, it should be written using HTML instead of markdown.